### PR TITLE
Export keystore file to google drive

### PR DESCRIPTION
**Pull request by adonayvasquezz**
[issue-72](https://github.com/akroma-project/akroma-wallet-mobile/issues/72)

## Description
- Users can export their wallet keystore file to their Google drive.

## To test
  1. Go to wallet
  2. Press the top right button.
  3. Select Wallet Settings.
  4. Press the top right button.
  5. Select export
  6. Select Export to Google drive button

Captura de Pantalla 2022-12-13 a la(s) 10 33 43

### CODE REVIEW

**Comment by detroitpro** [comment]
@Argude01 @adonayvasquezz - can you see why this failed on Bitrise.

I believe that once we have all the deployment/profiles/keys/assets in place - that bitrise builds should not fail unless we change something core.

**Comment by adonayvasquezz** [comment]
> @Argude01 @adonayvasquezz - can you see why this failed on Bitrise.
> 
> I believe that once we have all the deployment/profiles/keys/assets in place - that bitrise builds should not fail unless we change something core.

Ok, yes something core changed, let me check

File: src/screens/home/WalletSettingsScreen.tsx [hunk 1]
```diff
@@ -52,21 +55,50 @@ export const WalletSettingsScreen = ({ route }: { route: any }) => {
...
+  const exportToGoogleDrive = async () => {
+    const fileName = getKeystoreFileName();
+    const platformSetup = Platform.OS === 'android' ? { scopes: ['https://www.googleapis.com/auth/drive.file'] } : { iosClientId: '507652349383-rmhtbabce3ir7tk5pmv32j9qug0bfc9o.apps.googleusercontent.com' };
+
+    GoogleSignin.configure(platformSetup);
+    await GoogleSignin.signIn();
+
+    const gdrive = new GDrive();
+    gdrive.accessToken = (await GoogleSignin.getTokens()).accessToken;
+    //console.log('Lista en drive ', await gdrive.files.list());
```
Comment (cafadev) [inline src/screens/home/WalletSettingsScreen.tsx:87]: I think we can remove this commented code!

File: src/screens/home/WalletSettingsScreen.tsx [hunk 2]
```diff
@@ -52,21 +55,50 @@ export const WalletSettingsScreen = ({ route }: { route: any }) => {
...
+        console.debug('file written');
       })
       .catch(err => {
         console.log(err.message);
       });
   };
 
+  const exportToGoogleDrive = async () => {
+    const fileName = getKeystoreFileName();
+    const platformSetup = Platform.OS === 'android' ? { scopes: ['https://www.googleapis.com/auth/drive.file'] } : { iosClientId: '507652349383-rmhtbabce3ir7tk5pmv32j9qug0bfc9o.apps.googleusercontent.com' };
```
Comment (cafadev) [inline src/screens/home/WalletSettingsScreen.tsx:80]: Can we set an `env` variable to store the `iosClientId`?

**Review by cafadev — CHANGES_REQUESTED** [review]
Nice job on this one @adonayvasquezz. Just asking you for small changes to make

File: src/screens/home/WalletSettingsScreen.tsx (hunk 2 above)
Comment (adonayvasquezz) [reply src/screens/home/WalletSettingsScreen.tsx:80]: Yep, I'll setup the env variable

Comment (adonayvasquezz) [reply src/screens/home/WalletSettingsScreen.tsx]: due the client id is not a security issue i put it in a constants file

**Review by cafadev — APPROVED** [review]

**Pull request merged**